### DISCUSSION TITLE
[help] How can I prevent backslash `\` printed as `^H`?

### DISCUSSION THREAD

**Discussion by avatar-lavventura** [dandavison/delta#1419] [General]
I am working on a latex file where I have a lot of back slash character followed by `b` like `\bash{string}` or `\bottomrule`.

delta shows `\b` as `^H` :

<img width="575" alt="My Screenshot 2023-05-19 at 19 43 34" src="https://github.com/dandavison/delta/assets/18537398/34174a3c-2c10-4299-86c2-f0be15ea96a6">

I was not able to figure out cause of this error. Is it possible to prevent this?

my config:

```bash
[delta] 
        pager = less +A
        navigate = true
        light = false
        syntax-theme = Dracula
        line-numbers = false
        side-by-side = false
        minus-style = syntax "#3f0001"
        plus-style = bold syntax "#003800"
        whitespace-error-style = 22 reverse
        hunk-header-decoration-style = omit
        features = line-numbers highlight-removed side-by-side diff-so-fancy 
        markEmptyLines = false
        commit-style = yellow
        whitespace-error-style = reverse magenta
        file-style = omit
```

**Comment by dandavison** [comment]
Hi @avatar-lavventura, can you provide the raw input to delta that causes the problem? E.g. `git diff > /tmp/diff.txt` and paste the contents of `/tmp/diff.txt` into this issue.

**Comment by avatar-lavventura** [comment]
Sure: 

```
diff --git a/ama/eblocbroker.pdf b/ama/eblocbroker.pdf
index d532014..ab6ae53 100644
Binary files a/ama/eblocbroker.pdf and b/ama/eblocbroker.pdf differ
diff --git a/ama/eblocbroker.tex b/ama/eblocbroker.tex
index 5bbfd6c..5104564 100644
--- a/ama/eblocbroker.tex
+++ b/ama/eblocbroker.tex
@@ -872,7 +872,7 @@ The \bash{brokerd} performs the following steps:
-        %
+        % %
```

**Comment by dandavison** [comment]
This works for me. Does it work for you if you disable paging (e.g. `git diff | delta --paging=never`)?

<img width="657" alt="image" src="https://github.com/dandavison/delta/assets/52205/6e90930e-f823-4a38-9edd-9e18988462cd">

**Reply by avatar-lavventura** [reply]
`git diff | delta --paging=never` trims `\b` , I just see: `ash{alper} alper`

**Comment by avatar-lavventura** [comment]
Could you have a look into :

```
diff --git a/ama/new.tex b/ama/new.tex
index 06f0d3d..07b18a7 100644
--- a/ama/new.tex
+++ b/ama/new.tex
@@ -1 +1,2 @@
-\bash{alper}
+\bash{alper} alper
+%%
```

The output I have:

<img width="727" alt="My Screenshot 2023-05-19 at 22 36 54" src="https://github.com/dandavison/delta/assets/18537398/cd6be286-15f0-4850-9a52-883b9c83631a">

**Comment by dandavison** [comment]
<img width="554" alt="image" src="https://github.com/dandavison/delta/assets/52205/a403f6d8-3953-4974-b2ed-8a1e37084159">

**Comment by dandavison** [comment]
I think this is a problem with your pager (less) or your terminal emulator. Does the bug occur if you are not using a pager?

**Reply by avatar-lavventura** [reply]
I do :-(  // For example : `echo "\bash" | command delta` shows `^Hash`

**Reply by dandavison** [reply]
What about `git diff | cat`?

**Reply by avatar-lavventura** [reply]
Looks fine:

```bash
$ git diff | cat
diff --git a/ama/new.txt b/ama/new.txt
index 06f0d3d..0811a92 100644
--- a/ama/new.txt
+++ b/ama/new.txt
@@ -1 +1 @@
-\bash{alper}
+\bash{alper} is sad
```

**Reply by avatar-lavventura** [reply]
Not sure how but `git diff --color-moved=dimmed_zebra -w --ignore-blank-lines -U0 --ignore-space-at-eol HEAD | ifne delta --minus-style 'syntax bold auto' | ifne tail -n +2 | delta` works !
